packages/react/package.json

```diff
@@ -54,6 +54,8 @@
     "pluralize": "^8.0.0",
     "react-fast-compare": "^3.2.2",
     "react-hook-form": "~7.48.2",
+    "cmdk": "^1.0.4",
```

Comment:
Do we actually import stuff from cmdk, or just rely on components that use it under the hood?

Reply:
I don't think we need this actually